### ISSUE TITLE
TPP Not accepting liquids [1.6.0]

### ISSUE THREAD

**Issue by Ozimundis** [FTBTeam/FTB-Modpack-Issues#5583]
### Modpack

FTB NeoTech

### Modpack version

1.6.0

### Has the pack been modified

No

### Describe the bug

Similar to #5472 The TPP Kerosine>Gasoline and LPG+Sulfur recipies are playing up. 
In a fresh install and new instance of the pack and world 1.6.0, Kerosine>Gasoline + Gas>LPG do not work, however, LPG+Sulfur does. 

Dropping my old world save into the saves folder of the new instance, the recipies still seem to be broken, including LPG+Sulfur. 
At least the T.P.P will not accept new liquid to the input tank, any of aforementioned liquid will process, but new Kerosine/Gasoline will not be accepted.
Have tried With P:R small tanks and dispenser upgrades, M.I. Fluid cables and Immersive fluid cables (being fed from a fluid silo with redstone applied) and Mek small tanks + MI fluid pipes as well as Mek fluid cables directly.

The fact that exsisting fluids will be processed makes me think this is a mod issue, rather than recipies.

My old world instance was modded with the Decorative blocks mod added to the pack, but this new instance is "Stock"

### Steps to reproduce

Make/Place T.P.P and give appropriate pressure/heat, try to add recipie fluids, machine does not accept.

### Expected behaviour

TPP to accept fluids and process recipies

### Screenshots

![2024-08-01_16 23 31](https://github.com/user-attachments/assets/a35c36aa-1dc8-4f0c-90fe-bf8e96034cd5)
Screenshot of the new world showing TPP accepting and processing LPG>raw biofuel 

![2024-08-01_16 25 15](https://github.com/user-attachments/assets/077a0c0c-c9e8-4568-b2e9-34ef4322a71a)
Screenshot of old world setup, with TPP not accepting LPG by any method.

### Additional information

May be unrelated, but lastnight had a random CTD of the pack, including link to crash report, just to be thorough.  https://pste.ch/pewazebita.coffeescript 

Otherwise, not sure what else to add to help diagnose the issue.

**Comment by liafalk** [comment]
I've been struggling with this issue too, and I can confirm it works in a new world, but not my existing one (upgraded from NeoTech 1.4.2)

**Comment by Ozimundis** [comment]
![2024-08-01_17 11 25](https://github.com/user-attachments/assets/1ea414f6-b995-41e5-b123-00d6d6db94de)
 Additional post to show TPP with LPG and tank with dispenser upgrade but not pushing to TPP

**Comment by liafalk** [comment]
I somehow fixed this by making a new world on my server and changing back to my original world.

**Comment by DriftinFool** [comment]
I'm having the same issue. But it's extremely strange. Virulent mixture didn't work when I first set it up. Then the next day when I logged on, it was magically working. I proceeded to setup a pneumaticraft refinery turning kerosene and gasoline into LPG, then raw biodiesel, and it was working. Now today, none of the TPP's will accept fluids. I've tried buckets, tanks, pipes, and laser IO. And all of this is on the latest version. It works one day and then quits the next.

**Comment by xxhertz** [comment]
still an issue

**Comment by Saereth** [comment]
@xxhertz  or @gav1tron I reopened the issue because I could no longer reproduce this myself, any chance of you guys could zip your world and send it over to me?

**Comment by xxhertz** [comment]
sure thing, note this only happens randomly.. really inexplicable but it's on/off whenever i leave/join

might be laggy because my sfm program is really suboptimal fyi

also i have made these modifications to the pack (although they shouldnt matter)

\+ essential mod
\- minetogether mod
\- simple weather mod (was causing a crashloop mysteriously)

[the world](https://mega.nz/file/6VcQgADS#okOPpZ-y1HNbpkPXV8EBihpBUf5ee6I0tZ_Y9y8Hmn4)

apologies in advance if there's anything offensive, i was playing the pack with some friends

**Comment by xxhertz** [comment]
it seems like this isn't really happening in my setup anymore, but when i had purified virulent mix being output to ~8 of them it would mysteriously just stop working. i straight up eradicated this setup because i was so frustrated with how poorly it was functioning; all of the pneumaticcraft stuff is on the west side of my base in a 1x2 hole (-9291, 55, -17862)

i ended up using multiple logistics mods to try and fix it, which is why you see me using mekanism's pumps along with sfm pipes running right next to some of the machines

it's definitely still an issue, but once you get going it just seems to disappear or no longer be a problem at some point

**Comment by Saereth** [comment]
Try adding this mod and see if it resolves your instance. I've used it with your world and everything is working. I recreated some new TPP setups and reloaded the save a few times with no breakage. If this works for you I'll get it in the next pack update. https://maven.breakinblocks.com/releases/com/breakinblocks/pncfluidpatcher/pncfluidpatcher/1.20.4-1.0.0/pncfluidpatcher-1.20.4-1.0.0.jar

edit: I've spoken wish Desht, the Dev for PNC and I've submitted a PR to fix it in the mod he'll look over and hopefully get merged in to fix this without the mod but until that happens this should work for now
